README.md

```diff
@@ -38,4 +48,4 @@ https://groups.google.com/forum/#!forum/gonum-dev
 
 Original code is licensed under the Gonum License found in the LICENSE file. Portions of the code are subject to the additional licenses found in THIRD_PARTY_LICENSES. All third party code is licensed either under a BSD or MIT license.
 
-Code in graph/formats/dot is dual licensed [Public Domain Dedication](https://creativecommons.org/publicdomain/zero/1.0/) and Gonum License, and users are free to choose the license which suits their needs for this code.
\ No newline at end of file
+Code in graph/formats/dot is dual licensed [Public Domain Dedication](https://creativecommons.org/publicdomain/zero/1.0/) and Gonum License, and users are free to choose the license which suits their needs for this code.
```

Comment:
I take it that this is just an EOL addition.

Reply:
yes.
I can remove this if you prefer (I figured the amount of noise would be rather minimal).